### PR TITLE
add jhove install and remove jars from source

### PR DESCRIPTION
**Pull request by pwinckles**
Removes the jhove jars from the source tree and adds the setup that pulls in the jars as part of `generate-resources`. Note, this also upgrades from 1.26.0 to 1.26.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ installed.
 exiftool is installed using the script at `src/main/script/ExiftoolInstall` and is configured using the properties file
 `tools.properties`.
 
-Tika is installed exclusively through its pom file `tika-pom.xml`. 
+Tika and JHOVE are installed exclusive through their pom files `tika-pom.xml` and `jhove-pom.xml` respectively.
```
Comment (awoods) [inline README.md:124]: `exclusive` -> `exclusively` ?

Comment (pwinckles) [reply README.md:124]: :( That's what happens when you have a commit backlog

File: tool-poms/jhove-pom.xml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    <parent>
+        <groupId>edu.harvard.huit.lts</groupId>
+        <artifactId>fits-parent</artifactId>
+        <version>1.6.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <build>
+        <plugins>
+            <!-- Remove all JHOVE jars currently in the JHVOE dir -->
```
Comment (awoods) [inline tool-poms/jhove-pom.xml:21]: typo: `JHVOE` -> `JHOVE`

**Review by awoods — APPROVED** [review]
Looks good... will merge once the build passes.

**Pull request merged**